includes/admin/class-wcmp-export.php

```diff
@@ -370,6 +370,10 @@ public function add_shipments(array $order_ids, bool $process)
         }
 
         foreach ($order_ids as $order_id) {
+            if ($this->errors[$order_id]) {
```

Comment:
Als $order_id niet bestaat in errors, krijg je een error. Graag even keyexist, ! empty of isset gebruiken

Reply:
done